### PR TITLE
[bn] Localize portability.md & idempotence.md

### PR DESCRIPTION
**Pull request by AbdullahArean**
Signed-off-by: Abdullah Arean  2019-917-795@student.cse.du.ac.bd 
Localizes “content/bn/portability.md"  "content/bn/idempotence.md" to Bengali                    
This is an initial PR as may need some improvements!                     
Tracking issue: #410

### CODE REVIEW

File: content/bn/portability.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: বহনযোগ্যতা (Portability) 
+status: Completed
+category: সম্পত্তি 
+---
+
+বহনযোগ্যতা হল একটি সফ্টওয়্যার বৈশিষ্ট্য, পুনঃব্যবহারযোগ্যতার একটি রূপ যা নির্দিষ্ট অপারেটিং পরিবেশে "আটকে যাওয়া" এড়াতে সাহায্য করে,
```
Comment (mitul3737) [inline content/bn/portability.md:7]:
```suggestion
বহনযোগ্যতা হল একটি সফ্টওয়্যার বৈশিষ্ট্য এবং পুনঃব্যবহারযোগ্যতার একটি রূপ যা নির্দিষ্ট অপারেটিং পরিবেশে "আটকে যাওয়া" এড়াতে সাহায্য করে ।
```

File: content/bn/portability.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+যেমন ক্লাউড প্রোভাইডার, অপারেটিং সিস্টেম বা এ সংশ্লিষ্ট কোম্পানির পরিবেশে।
```
Comment (mitul3737) [inline content/bn/portability.md:8]:
```suggestion
যেমনঃ অপারেটিং সিস্টেম বা এ সংশ্লিষ্ট কোম্পানির পরিবেশে ক্লাউড প্রোভাইডার ।
```

File: content/bn/portability.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+প্রথাগতভাবে, সফ্টওয়্যার প্রায়ই নির্দিষ্ট পরিবেশের জন্য তৈরি করা হয় (যেমন এডব্লিউএস বা লিন্যাক্স)।
+অন্যদিকে, বহনযোগ্য সফ্টওয়্যারগুলি বিভিন্ন অপারেটিং পরিবেশে কাজ করে বড় কোন পরিবর্তনের  প্রয়োজন ছাড়াই।
+একটি অ্যাপ্লিকেশন বহনযোগ্য বলে বিবেচিত হয় যদি এটিকে একটি নতুন পরিবেশের সাথে খাপ খাইয়ে নেওয়ার জন্য প্রয়োজনীয় প্রচেষ্টা যুক্তিসঙ্গত সীমার মধ্যে থাকে।
+"বহন করা" শব্দগুচ্ছের অর্থ হল সফ্টওয়্যার পরিবর্তন করা এবং এটিকে একটি ভিন্ন কম্পিউটার সিস্টেমে কাজ করার জন্য অভিযোজিত করা।
```
Comment (mitul3737) [inline content/bn/portability.md:13]:
```suggestion
"বহন করা (to port) " শব্দগুচ্ছের অর্থ হল সফ্টওয়্যার পরিবর্তন করা এবং এটিকে একটি ভিন্ন কম্পিউটার সিস্টেমে কাজ করার জন্য অভিযোজিত করা।
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by mitul3737 — APPROVED** [review]
lgtm

**Comment by mitul3737** [comment]
@AbdullahArean  kindly don't localize 2 files in one branch .

**Review by Imtiaz1234 — APPROVED** [review]
LGTM

**Comment by AbdullahArean** [comment]
It was by accident, whatever, I will try my best to follow best practices. Thanks @mitul3737

**Pull request merged**